jobs.groovy

```diff
@@ -675,26 +740,14 @@ def build_multijob(dashBranchName, branchName, isReleaseBuild) {
                Skip the cron jobs as they don't run from this multijob,
                so the copied artifacts could be from a build of a previous
                revision, which may confuse things. */
-            for (job_type_entry in GLOBAL_CONFIG.job_type.findAll { it.key != 'cronly_jobs' } ) {
-                job_type = job_type_entry.key
-                for (job_entry in job_type_entry.value) {
-                    job_name = job_entry.key
-                    job_values = job_entry.value
-                    for (_module in job_values.with_modules) {
-                        /* no every job produces an artifact, so make sure wew
-                           don't try to fetch artifacts for jobs that don't
-                           produce them */
-                        if (job_values.archive_artifacts) {
-                            _job_name = job_entry.key + '_' + escape_name(_module)
-                            copy_artifacts_from(
-                                delegate, full_job_name(dashProject, dashBranchName, _job_name),
-                                 _job_name, job_values.archive_artifacts
-                            )
-                        }
-                    }
+            list_jobs(dashProject, dashBranchName).findAll { it.job_type != 'cronly_jobs' }.each {
```

Comment:
Shouldn't this be `it.type` rather than `it.job_type`? More worryingly, shouldn't this cause something to fail / explode?